test/model/unreads_test.dart

```diff
@@ -469,6 +469,145 @@ void main() {
         }
       }
     });
+
+    group('moves', () {
+      final origChannel = eg.stream();
+      const origTopic = 'origTopic';
+      const newTopic = 'newTopic';
+
+      final readMessages = List<StreamMessage>.generate(10,
+        (_) => eg.streamMessage(
+          stream: origChannel, topic: origTopic, flags: [MessageFlag.read]));
+      final unreadMessages = List<StreamMessage>.generate(10,
+        (_) => eg.streamMessage(stream: origChannel, topic: origTopic));
```

Comment:
nit: make the parallel parts visibly parallel:
```suggestion
        (_) => eg.streamMessage(stream: origChannel, topic: origTopic,
                 flags: [MessageFlag.read]));
      final unreadMessages = List<StreamMessage>.generate(10,
        (_) => eg.streamMessage(stream: origChannel, topic: origTopic));
```